src/customers/account-dashboard-requisition-lists-manage.md

```diff
@@ -69,6 +63,29 @@ Magento will generate and download a CSV list with `sku` and `qty` values.
    ![]({% link images/images-b2b/requisition-list-print.png %}){: .zoom}
    _Print Requisition List_
 
+### Edit product options
+
+To edit a product options in the list do the following:
+
+1. Go to the product page via the **‘Pencil’** button. (![]({% link images/images/btn-edit.png %})).
+
+1. Change any options that are necessary.
+
+1. Click **Update Requisition List** button.
+
+   ![]({% link images/images-b2b/update-requisition-list.png %}){: .zoom}
+   _Update Requisition List_
+
+Product in the Requisition List may be edit if:
+
+1. Product has **all options set** (in case you have a [configured product]({% link catalog/product-create-configurable.md %}) in the Requisition List).
+
+1. Product is **added to this Requisition List**.
```

Comment:
Option is correct regarding to [core logic](https://github.com/magento/partners-magento2b2b/blob/934c1ebff9f2a3c7389aadd2b30cffe89f33d684/app/code/Magento/RequisitionList/Block/Requisition/View/Item.php#L363).